tools/detect_unused_modules

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+# checkout sle11 test plan to also check for modules used in there. sle11
+# should be the only other older testplan which might use test modules from
+# master
+git co origin/sle11_only_testplan -- main.pm
```

Comment:
yes, makes sense. Ok, let's wait for https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2096 and then we shouldn't need this anymore